kontakt-reservasjonsregister-client/src/main/java/no/difi/meldingsutveksling/ptp/KontaktInfo.java

```diff
@@ -7,6 +7,7 @@
     private final PersonKontaktInfoMapper.MailboxProvider providerDetails;
     private final PersonKontaktInfoMapper.PersonDetails personDetails;
     private PrintProviderDetails printDetails;
+    private boolean obligatedToBeNotified;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "obligatedToBeNotified" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1068)